quiz.html

```diff
@@ -11,7 +11,7 @@
 			<a href="index.html" class="logo">Skinclusive</a>
 			<nav name="top">
 				<ul>
-					<li><a href="quiz.html">skin type quiz</a></li>
+					<li><a href="quiz.html" class="current">skin type quiz</a></li>

```

Comment:
Website iteration where the nav link will underline depending on the page which they are currently on. So in this case, if the user is on the quiz page, then the link will be underlined.